eth/chains/mainnet/constants.py

```diff
@@ -42,3 +42,9 @@
 # Petersburg Block
 #
 PETERSBURG_MAINNET_BLOCK = BlockNumber(7280000)
+
+#
+# Istanbul Block
+#
+# FIXME: Replace this with the correct blocknumber when it's decided
+ISTANBUL_MAINNET_BLOCK = BlockNumber(7280728)
```

Comment:
I think this can now be changed to `9069000`.  The meta-EIP for Istanbul has been [updated](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1679.md#activation) with this change.

Reply:
Ah, this escaped me somehow. One sec...